res/adpods/vmap-truex.xml

```diff
@@ -0,0 +1,15 @@
+<vmap:VMAP xmlns:vmap="http://www.iab.net/videosuite/vmap" version="1.0">
```

Comment:
Just curious, but can RAF support being instantiated with the type of VAST in `truex-pod-preroll.xml` or `truex-pod-midroll.xml` instead?

The way the client described to us is they make a ad call right before every pod (I assume they hard code their pod break index or comes in a separate CMS?) and get back VAST. They explicitly stated not working with VMAP. 

So I was curious if there's a way to go directly to VAST or RAF simply had to be instantiated with VMAP.

I'm not asking for any change at this stage, as I think the example is readable even with VMAP, and can readily be adapted, just trying to understand in case there's pushback on the use of VMAP in the example...

Reply:
The vast file's can be passed directly (eg truex-pod-preroll.xml) and it will work.  I didn't have a clear understanding of what OTTera's ad layer was like.  But given that you can pass the pods immediately, I figured it will be fine to showcase the flow with vmap and then adjust from there.  (Also had some difficulty with the ad construction layer)

Your description does sound like a bit of a gap though.  If the client is going to make ad calls immediately, it makes part of the RAF logic a bit odd.  They would need to replace raf.getAds() with their own behaviour, along with some kind of ad position mechanism.  I suppose one can assume they should have some of this in place already, they just need to adjust it to support TrueX.....

Reply:
You are making good points. All things considered let's stick with the VMAP mechanism for this version and take it from here with the client.